CONTRIBUTING.md

```diff
@@ -1,41 +1,15 @@
+To get involved in making updates or translations for jamulus.io you will need a GitHub account and be familiar with GitHub in general. If you want to get more involved, you’ll need to be familiar with the use of git on the command line, as well as Jekyll. Note that translating the Jamulus application itself is done by a different process, but you are welcome to help with that too. Introduce yourself on the [GitHub Discussion](https://github.com/jamulussoftware/jamulus/discussions) and say what you’d like to do.
 
-If you want to open a Pull Request (after having described and discussed your proposal), keep in mind that your changes (if they are not just typos) might have to wait until everything is translated. We will bundle them (if everything is all right) with any other changes and upload all of them as a combined Pull Request to jamulus.io.
+## Want to add a new language?
 
-Pull requests/issues for the website should have a title which looks like this:
+Great! First [open an issue](https://github.com/jamulussoftware/jamuluswebsite/issues) with the title `[Translate] yourLanguage` and introduce yourself. We will then help you to set up the relevant language files. 
 
-`[Tag (like Bug, Translation, etc.)][language code (e.g. en)] {short description of your changes}`
-
-## Branches, Translation and development process
-
-We currently use *multiple branches* to ensure that translators know what to translate.
-
-After having discussed and agreed on changes you want to be integrated to jamulus.io, you should ask yourself some questions:
-
-## 0. Is your content in a language that already exists on the site?
-If not: This will be discussed this later (since you introduce a new language).
-If so look here:
-### 1. Be clear if your change needs translation
-**No:** A fix which doesn't need to be translated (e.g. typo) --> Open a Pull Request to `release`. It will go live a few minutes after the PR is merged
-**Yes:** A fix which needs to be translated (e.g. if you want to make a sentence or paragraph clearer) --> Commit to `changes` since others will need to translate it. (If you wish to, please do not add your translation to your commit, but add a comment with the translation in your Pull Request). It will be part of the next release.
-
-### 2. Be clear if a translation update is going on
-You can see this if we ping the translators in a PR from changes to translate.
-
-**If yes:** only translate the changes of the Pull Request from changes to translation. If you find issues, start with 1
-**If no:** no need to care about the translate branch
-
-## Adding a new language
-
-1. Open an issue with the title [Translate] `yourLanguage` and say that you want to translate the homepage into your language.
-2. If you start your new translation, follow the instructions to add a new language in the /README.md file on the changes branch.
-3. Finally, open a PR to `translate`
+### Please also note that discussions should be in English so as to be open to all. 
```

Comment:
Better don't abuse a md headline here. Maybe we can do it with a css class?

Reply:
Do we have one?

Reply:
Not sure. But maybe on the server page there's one (the one we use to format the NO)

Reply:
BTW this is page is in repo, not the GitHub Pages site, so I assume we only have markdown.